### ISSUE TITLE
Bookmark toolbar folder content reaction (left side versus right side)

### ISSUE THREAD

**Issue by rpgmaniac** [brave/brave-browser#13550]
This is what happen when you check the content of a folder you have on the left side https://i.ibb.co/wWx2K87/2021-01-13-191950.png & this is what happen when you check the content of a folder you have placed in the far right side https://i.ibb.co/7W96JP4/2021-01-13-192043.png & the question is why this is happening? this was not an issue in firefox, in Brave I can't place a folder in the right side as the visible content is very limited & that's very inconvenient, I would like to see a fix of this plz, this is faulty design & I can't comprehend what did you make this work like this.

**Comment by rebron** [comment]
@rpgmaniac Can you attach a full screenshot of your browser window? I suspect this reproduces in Chrome. I can't seem to reproduce this myself. Did you import your Firefox bookmarks? Could you try creating a new `Path of Exile` bookmark folder and putting your bookmarks in the new folder and see if that does it? Can experiment with a new folder with a few bookmarks and see if that does it too. 

I'm seeing the same menu width regardless of whether the folder is on the furthest right or left.

**Comment by rpgmaniac** [comment]
@rebron Yes I import my Firefox bookmarks, yes I tried to create a new folder & put my bookmarks there didn't change anything the result was exactly the same, if you wanna replicate it fill the toolbar with bookmarks & leave at the far right side enough space just for the folder you wanna put there, this is when this thing happen & as I said this was not the case in Firefox.

**Comment by rebron** [comment]
I'm following your steps and seeing full width. I'm suspecting it's an issue with one of the bookmarks in that folder itself that's screwing up the formatting.

You're saying a totally new folder in the far right with a different set of bookmarks exhibits the same behavior? I'm seeing full width properly in a couple scenarios. 

Can you also go to brave://version and copy and paste the version browser you've got?

Brave | 1.21.2 Chromium: 88.0.4324.79 (Official Build) nightly (arm64)
-- | --
Revision | bd1e9353659b2491dac971226a973ca3b5684a14-refs/branch-heads/4324@{#1520}
OS | macOS Version 11.1 (Build 20C69)

**Comment by rpgmaniac** [comment]
@rebron https://i.ibb.co/64YGPNH/2021-01-15-083744.png



> You're saying a totally new folder in the far right with a different set of bookmarks exhibits the same behavior?

Yep, I tried few minutes ago to create a new folder & move all the bookmarks there & didn't help, the same thing happen.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
